src/convertToObject.js

```diff
@@ -3,16 +3,44 @@
 /**
  * Implement convertToObject function:
  *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
+ * Function takes a string with styles
+ * (see an example in [stylesString.js](./stylesString.js))
  * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
+ * and values are the values of related CSS properties
+ * (see an exampl in [test file](./convertToObject.test.js))
  *
  * @param {string} sourceString
  *
  * @return {object}
  */
+
 function convertToObject(sourceString) {
-  // write your code here
+  const stylesArray = sourceString.split('\n');
+  const trimmed = [];
+
+  stylesArray.forEach((style) => {
+    trimmed.push(style.trim().replace(';', ''));
+  });
+
+  const cleared = [];
+
+  trimmed.forEach((e) => {
```

Comment:
You are using a forEach loop to create a new array. A more efficient and readable way to do this would be using the map function.